dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -309,7 +309,7 @@ def run_validation(self, job):
 
             error_list.write_all_row_errors(job_id)
             # Update error info for submission
-            populate_submission_error_info(submission_id)
```

Comment:
I reused it actually in `run_cross_validation` since it was actually duplicating the code